pkg/util/ip_reservation.go

```diff
@@ -103,7 +94,7 @@ func (ipAllocator *IPAllocator) GetUnreservedIPRange(cidr string, cloudInstances
 		reservedIPRanges[reservedIPRange] = true
 	}
 
-	for cidrIP := cloneIP(ip.Mask(ipnet.Mask)); ipnet.Contains(cidrIP) && err == nil; cidrIP, err = incrementIP(cidrIP, incrementStep29IPRange) {
+	for cidrIP := cloneIP(ip.Mask(ipnet.Mask)); ipnet.Contains(cidrIP) && err == nil; cidrIP, err = incrementIP(cidrIP, (byte)(math.Exp2(float64(ipV4Bits-ipRangeSize)))) {
```

Comment:
To make it self-commenting, pull out incrementStepIPRange = (byte)(math.Exp2(float64(ipV4Bits - ipRangeSize))